src/iuse.cpp

```diff
@@ -3064,7 +3064,7 @@ int iuse::radio_on(player *p, item *it, bool t, const tripoint &pos)
             }
 
             std::vector<std::string> segments = foldstring(message, RADIO_PER_TURN);
-            int index = calendar::once_every(segments.size());
+            int index = calendar::get_radio_message_index(segments.size());
```

Comment:
I'm all in favor of putting things into separate function, but this seems excessive. The new function is used exactly once, it's specific to that situation. Why not move the `turn % size()` part directly into this line?